guides/INSTALLING.md

```diff
@@ -27,152 +47,252 @@ As listed step-by-step in the [Getting Started](https://github.com/ONSdigital/dp
     - Elasticsearch 5 (on non-standard port)
     - Kafka (plus required Zookeeper dependency)
     - Neo4J
-* [go v1.13](https://golang.org/doc/install)
 * [GoConvey](https://github.com/smartystreets/goconvey#installation)
-* [Govendor](https://github.com/kardianos/govendor)
 * [GhostScript](https://www.ghostscript.com/download.html) - Required for [Babbage](https://github.com/onsdigital/babbage)
-* [Vault](https://www.vaultproject.io/intro/getting-started/install.html) - This will be required for running Florence.
 
-* [jq](https://stedolan.github.io/jq/) - installed with `brew install jq`
-  - This isn't essential, but it's a useful tool for debugging website content.
+  - `$ brew install ghostscript`
+* [Vault](https://www.vaultproject.io/intro/getting-started/install.html) - Required for running Florence.
+
+  - `$ brew install hashicorp/tap/vault`
+
+* [jq](https://stedolan.github.io/jq/) - a handy JSON tool (for debugging website content)
+  - `$ brew install jq`
+
+* [yq](https://github.com/mikefarah/yq) - a handy YAML tool
+  - `$ brew install yq`
 
 Return to the [Getting Started](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md) guide for next steps.
 
---------------
+---
 
 ## Clone the services
 
-Clone the GitHub repos for [web](#web), [publishing](#publishing) and/or [CMD](#cmd). Unless otherwise stated, services are run using `$ make debug`
+Clone the GitHub repos for [web](#web-journey), [publishing](#publishing-journey) and/or [CMD](#cmd-journeys).
 
-[Web](#web) will be enough strictly to work on website content types other than filterable datasets (e.g. bulletins, articles, timeseries, datasets). 
+- [Web](#web-journey) - These apps make up the public facing website providing read only access to published content, and will be enough strictly to work on website content types other than filterable datasets (e.g. bulletins, articles, timeseries, datasets). 
 
-[Publishing](#publishing) gives you the ability to update, preview and publish content - required for changes to any part of the publishing system. 
+- [Publishing](#publishing-journey) - The "publishing journey" gives you all the features of web together with an internal interface to update, preview and publish content. All content is encrypted and requires authentication.
 
-[CMD](#cmd) apps will support the filterable dataset journey, and would mean you have ever possible service running.
+- [CMD](#cmd-journeys) - apps will support the filterable dataset journey, and would mean you have every possible service running.
 
```

Comment:
[improvement] I think it would be worth mentioning the dp-tool alternative (the feature that @gedge demoed in the show and tell) here, in order to clone all the required repositories for each journey with a single command?

Reply:
that's not released yet, so not ready for here - maybe after DAC 😢

Reply:
yes leave for later